### PR TITLE
Update plugin for response body listing and Docusaurus 2 v0.70+

### PR DESCRIPTION
**Pull request by avaidyam**
(Based on work by @appaquet -- thank you!)

I'd also like to also make the following minor changes: 
- [ ] add the operation ID to each operation's page
- [ ] add a better authentication popup dialog (with support for multiple auth types)
- [ ] add server address variable support.
- [ ] use `x-codeSamples` load samples instead of the default `Node, Go, Python` ones in the demo panel.
- [ ] show all models under a single category/tag

@bourdakos1 Preliminary thoughts on merging this?

### CODE REVIEW

File: docusaurus-plugin-openapi/src/openapi.ts [hunk 1]
```diff
@@ -40,7 +40,12 @@ function getPaths(spec: OpenApiObject): ApiItem[] {
         if (isOperationObject(val)) {
           let method = key;
           let operationObject = val as OperationObject;
-          const summary = operationObject.summary || "Missing summary";
+
+          const summary = operationObject.summary || operationObject.operationId || "Missing summary";
+          if (!operationObject.description) {
```
Comment (bourdakos1) [inline docusaurus-plugin-openapi/src/openapi.ts:45]: We could probably use nullish coalescing here “??” Instead of “||”

**Review by bourdakos1 — COMMENTED** [review]
This looks pretty good to me, I’ll take a closer look when I get home

**Comment by appaquet** [comment]
@avaidyam You're welcome 😅 ! I wanted to upstream these changes, but just didn't find the time back then and it slipped through the cracks.

File: docusaurus-plugin-openapi/package.json [hunk 2]
```diff
@@ -52,10 +52,13 @@
     "redux": "^4.0.5",
     "redux-devtools-extension": "^2.13.8",
     "remark-admonitions": "^1.2.1",
-    "traverse": "^0.6.6"
+    "traverse": "^0.6.6",
+    "@docusaurus/core": "2.0.0-alpha.69",
```
[the remark this answers is not in the document]
Comment (avaidyam) [reply docusaurus-plugin-openapi/package.json:56]: I think it should be a `peerDependency` but for some reason it was set to `^2.0.0` and that made NPM v7.6.2 complain and refuse to install the original package. (I eventually forced it using `npm i --legacy-peer-deps` but that's not a great long-term strategy I reckon.)

**Comment by avaidyam** [comment]
@appaquet No problem! Your branch had the most progress towards what I wanted to see in this plugin, so I figured it was a great place to start. 😄

File: docusaurus-plugin-openapi/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (bourdakos1) [reply docusaurus-plugin-openapi/package.json:56]: Oh right I heard there were a lot of issues with peer deps with the new npm

**Review by bourdakos1 — APPROVED** [review]

**Pull request merged**